### ISSUE TITLE
NTR: Triggered by volatile organic compounds (Chest, respiratory & breathing)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7796]
New term request Triggered by volatile organic compounds (segal_220817170747): VOCs, odors, odours

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000180
- label: Triggered by volatile organic compounds
- parents: [Triggered by (HP:0025204)](https://hpo.jax.org/app/browse/term/HP:0025204)
- pmid: PMID:34948756
- def: Applies to a sign or symptom that is provoked or brought about by exposure to volatile organic compounds (VOCs), which comprise various organic chemicals which are released as gases from different liquids or solids.
- comment: VOCs are present in different household, industrial or commercial and products, but their accumulation in air and water has primarily gained attention. Among VOCs, trichloroethylene and vinyl chloride are the most toxic and carcinogenic compounds.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7796
- synonyms:
